data_structures.md

```diff
@@ -187,7 +187,7 @@ True
 
 **How It Works**
 
-The example is pretty much self-explanatory because it involves basic set theory mathematics taught in school.
+If you remember basic set theory mathematics from school, then this example is fairly self-explanatory.  But if not, I�d encourage you to google this topic as a general understanding in set theory and the Venn diagram will help you make better use of sets in Python.
```

Comment:
should we say:

> google "set theory" and "venn diagram"

Reply:
Yes, I agree- I'll make the change and commit it.